src/components/Book.js

```diff
@@ -19,7 +25,7 @@ Book.propTypes = {
   id: PropTypes.string.isRequired,
```

Comment:
- [ ] Take a look at the console log issue below. You are type checking for string whereas your id is a **`number`** (`Date.now()`). Kindly fix this

  ![mithi-prop](https://user-images.githubusercontent.com/36542986/150133333-56ef1f49-d87c-486b-aea8-3316eb7ed62d.jpg)